### PR TITLE
Update nemotron_340b.yaml

### PR DESCRIPTION
**Pull request by zhenghax**
Update the recipe arguments to achieve similar MFU of paper https://arxiv.org/pdf/2406.11704

### CODE REVIEW

**Comment by pablo-garay** [comment]
@dimapihtar could you PTAL @ this PR ?

File: launcher_scripts/conf/training/nemotron/nemotron_340b.yaml [hunk 1]
```diff
@@ -188,4 +196,4 @@ model:
       - .0333
       - ${data_dir}/my-nemotron_00_text_document
       - .0333
-      - ${data_dir}/my-nemotron_00_text_document
\ No newline at end of file
+      - ${data_dir}/my-nemotron_00_text_document
```
Comment (pablo-garay) [inline launcher_scripts/conf/training/nemotron/nemotron_340b.yaml:199]: why is this line added here needed ?

**Review by dimapihtar — APPROVED** [review]
LGTM. Thank you!
